### ISSUE TITLE
Remove use_next_extras from adder.

### ISSUE THREAD

**Issue by DriesSmit** [instadeepai/Mava#198]
Update the environment_loop function to still allow for the action extra information (e.g. logits) through the environment. 
Check that MA-DDPG and MA-PPO still run using the new adder architecture.
